akka-persistence/src/main/scala/akka/persistence/Persistence.scala

```diff
@@ -105,8 +105,18 @@ trait PersistenceRecovery {
    * To skip recovery completely return `Recovery.none`.
    */
   def recovery: Recovery = Recovery()
-  //#persistence-recovery
 }
+//#persistence-recovery
+
+//#persistence-stash
```

Comment:
this special comment is used from documentation includes, but not needed for this